### PR TITLE
Install.rst: Add note for ubuntu users

### PR DESCRIPTION
**Pull request by vaishnavm217**
Adds note for not using apt for installing coala.

Closes https://github.com/coala/documentation/issues/392

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ x ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ x ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by rohanraju97** [comment]
Change the commit body to something better.
"Adds note to not use 'apt-get' to install coala"

File: Users/Install.rst [hunk 1]
```diff
@@ -60,6 +60,9 @@ system-wide, use:
     For this and all future steps, some steps require root access
     (also known as administrative privileges in Windows).
 
+    **Ubuntu Users** - Don't use ``apt-get install coala``
+    for installing coala as that is different software.
```
Comment (satwikkansal) [inline Users/Install.rst]: Wrong place to insert this note as it breaks the context b/w the above and below lines :sweat_smile:

**Comment by satwikkansal** [comment]
Also enclose "apt" into single ticks `` in the commit body :)

File: Users/Install.rst (hunk 1 above)
Comment (vaishnavm217) [inline Users/Install.rst]: should i write above 
```For this and all future steps, some steps require root access (also known as administrative privileges in Windows).``` ? That seems apt place to put this

**Comment by vaishnavm217** [comment]
wait is it single ` or ``?

**Comment by satwikkansal** [comment]
Double ticks

**Comment by jayvdb** [comment]
In your commit message, ubuntu should be Ubuntu , but that is very minor.

**Comment by rohanraju97** [comment]
ack 2c06f56bb1953c93c2cd47cc279f2bb332041c5d

**Comment by jayvdb** [comment]
@rultor merge

**Comment by vaishnavm217** [comment]
YAY! finally :)

**Pull request merged**